compressor/management/commands/compress.py

```diff
@@ -10,7 +10,12 @@
 import django.template
 from django.template import Context
 from django.utils import six
-from django.utils.datastructures import SortedDict
+
+try:
+    from collections import OrderedDict
+except ImportError:
+    from django.utils.datastructures import SortedDict as OrderedDict
```

Comment:
Well, yes, that goes with Django 1.4 support. 

It would be good to pull out all/any compat code when we do that, but it's more than I have time for at this exact moment. :-)